pom.xml

```diff
@@ -27,6 +27,7 @@
   <modules>
     <module>bunsen</module>
     <module>pipelines</module>
+    <module>coverage</module>
```

Comment:
Why do we need a separate module just for coverage reporting? I have seen multi-module projects that use CodeCov and I don't think they need to do this trick. As an example, [openmrs fhir2](https://github.com/openmrs/openmrs-module-fhir2) has multiple modules and it seems they are properly reported in CodeCov [here](https://app.codecov.io/gh/openmrs/openmrs-module-fhir2).

Also when I look at the [CodeCov report for this PR](https://app.codecov.io/gh/google/fhir-data-pipes/pull/838) the numbers reported for the incremental coverage ("Change" column) does not seem to be right since this PR does not change any code.